maestro-test/src/test/resources/102_graaljs_dangerous_env.yaml

```diff
@@ -0,0 +1,7 @@
+appId: com.example.app
+jsEngine: graaljs
+env:
+  MAESTRO_CLI_DANGEROUS_GRAALJS_ALLOW_ALL_ACCESS: true
```

Comment:
We must not allow any variables that start with `MAESTRO_CLI` (even more so with `MAESTRO_CLI_DANGEROUS` to be set inside flow files.

(if my thinking is wrong please tell me)